cogs/audio.py

```diff
@@ -923,6 +985,20 @@ def is_playlist_name_valid(self, name):
             result = {"title": "A song "}
         return result
 
+    def chop(self, ctx, stuff:str):
+        message = stuff
+        if message == None:
+            message = ctx.message.author.name
+        else:
+            message = ctx.message.clean_content
+            message = message[(len(ctx.prefix)+len(ctx.command.name))+1:]
```

Comment:
I would be careful of using this.
I believe it bugs out if a prefix is a mention.
I recommend just going through `stuff` and converting the mentions in that.